src/net/java/sip/communicator/service/protocol/media/CallPeerMediaHandler.java

```diff
@@ -1595,15 +1595,16 @@ protected boolean requestKeyFrame()
      * @param stream <tt>MediaStream</tt> non-null stream
      * @param mediaType <tt>MediaType</tt>
      */
-    protected void sendHolePunchPacket(MediaStream stream, MediaType mediaType)
+    public void sendHolePunchPacket(MediaStream stream, MediaType mediaType)
```

Comment:
why?

Reply:
Really don't remember this edit!!!

Reply:
ah @ibauersachs I merged upstream to my origin. Seems soething was wrong. WIll checj

Reply:
@ibauersachs this is the master
https://github.com/jitsi/jitsi/blob/master/src/net/java/sip/communicator/service/protocol/media/CallPeerMediaHandler.java#L1598
So the issue is: this pull request contains merge commits.

Reply:
I'll close it and create new one with one huge commit

Reply:
No, please don't. I can't review 100+ files again.

Reply:
> @ibauersachs this is the master
> https://github.com/jitsi/jitsi/blob/master/src/net/java/sip/communicator/service/protocol/media/CallPeerMediaHandler.java#L1598
> So the issue is: this pull request contains merge commits.

I don't know why this shows up in the diff then, but okay, doesn't matter then.

Reply:
@ibauersachs please check once you have time